torch/nn/modules/loss.py

```diff
@@ -297,6 +297,59 @@ def forward(self, log_input: Tensor, target: Tensor) -> Tensor:
                                   eps=self.eps, reduction=self.reduction)
 
 
+class GaussianNLLLoss(_Loss):
+    r"""Negative log likelihood loss with Gaussian distribution
+
+    For targets modelled as having Gaussian distribution:
+    .. math::
+    target \sim \mathcal{N}(input, var)
```

Comment:
sum?

Reply:
Using sim to mean the target is Gaussian distributed with mean 'input' and variance 'var'.